src/main/java/com/artipie/http/PromuSlice.java

```diff
@@ -51,41 +53,51 @@ public PromuSlice(final Metrics metrics) {
     @Override
     public Response response(final String line, final Iterable<Map.Entry<String, String>> headers,
         final Publisher<ByteBuffer> body) {
-        final String ctype = new Accept(headers).values().get(0);
-        final StringWriter writer = new StringWriter();
-        final Set<String> filters = PromuSlice.params(line, "name");
-        final MetricsOutput output = new PrometheusOutput(writer, ctype, filters);
-        this.metrics.publish(output);
-        return new RsFull(
-            RsStatus.OK,
-            new Headers.From(new ContentType(ctype)),
-            new Content.From(
-                String.valueOf(writer.getBuffer()).getBytes(StandardCharsets.UTF_8)
-            )
-        );
+        final String ctype = new RqHeaders(headers, Accept.NAME).get(0);
```

Comment:
Accept header may contain multi mime-type value with (or without) qualifiers, e.g. 
```http
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,*/*;q=0.8
```
So you need to get the top mime-type here (like it was done previously: `new Accept(headers).values()`).

See https://developer.mozilla.org/en-US/docs/Web/HTTP/Content_negotiation/List_of_default_Accept_values

Reply:
Ok